includes/editor-templates/panel.php

```diff
@@ -70,13 +70,12 @@
 		<i class="elementor-icon eicon-menu-bar tooltip-target" aria-hidden="true" data-tooltip="<?php esc_attr_e( 'Menu', 'elementor' ); ?>"></i>
 		<span class="elementor-screen-only"><?php echo esc_html__( 'Menu', 'elementor' ); ?></span>
 	</button>
-	<# if ( $e.components.get( 'document/elements' ).utils.allowAddingWidgets() ) { #>
 	<h2 id="elementor-panel-header-title"></h2>
-	<button id="elementor-panel-header-add-button" class="elementor-header-button">
+	<# const extraClass = $e.components.get( 'document/elements' ).utils.allowAddingWidgets() ? '' : 'elementor-visibility-hidden'; #>
```

Comment:
Why are you now just hiding it instead of not rendering it?

Reply:
@louiswol94 as above, I was hiding one thing that I didn't want to hide, the `<h2>` . But to keep the h2 without breaking the layout I need to use the visibility thing, because a visibility:hidden element interacts with flex differently than display:none element